src/seals/testing/envs.py

```diff
@@ -243,6 +244,48 @@ def test_premature_step(env: gym.Env, skip_fn, raises_fn) -> None:
         env.step(act)
 
 
+def test_render(env: gym.Env, raises_fn) -> None:
+    """Test that render() supports the modes declared.
+
+    Example usage in pytest:
+        test_render(env, raises_fn=pytest.raises)
+
+    Args:
+        env: The environment to test.
+        raises_fn: Context manager to check exception is thrown.
```

Comment:
Should mention here the particular case in which `raises_fn` is used to catch which particular exception (`NotImplementedError`).